kirby.php

```diff
@@ -748,12 +775,12 @@ public function launch() {
     // set the timezone for all date functions
     date_default_timezone_set($this->options['timezone']);
 
-    // load all extensions
-    $this->extensions();
-
     // load all plugins
     $this->plugins();
 
+    // load all extensions
+    $this->extensions();
```

Comment:
The modules feature `tags` as its own modules type atm (as there is also `site/tags`), which is handled by the `extensions()` method. That's why we first have to load the plugins which could register the bundled tags. I thought about if it could break anything, but nothing came to mind - but it's def one of the things why we would need a lot of testing.